apiserver/facades/controller/charmrevisionupdater/updater_test.go

```diff
@@ -185,16 +185,18 @@ func (s *charmVersionSuite) TestJujuMetadataHeaderIsSent(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	cloud, err := s.State.Cloud(model.Cloud())
 	c.Assert(err, jc.ErrorIsNil)
-	expected_header := []string{
-		"environment_uuid=" + model.UUID(),
-		"model_uuid=" + model.UUID(),
-		"controller_uuid=" + s.State.ControllerUUID(),
+	expectedHeader := []string{
 		"cloud=" + model.Cloud(),
 		"cloud_region=" + model.CloudRegion(),
-		"provider=" + cloud.Type,
+		"controller_uuid=" + s.State.ControllerUUID(),
 		"controller_version=" + version.Current.String(),
+		"environment_uuid=" + model.UUID(),
+		"is_controller=true",
+		"model_uuid=" + model.UUID(),
+		"provider=" + cloud.Type,
+		"series=quantal",
```

Comment:
can we verify that multi-series charms don't behave in any odd way and I'm also curious with the ability to have a multi-series subordinate related to different series in the same model how that would effect this. 

If the data's not perfect that's ok but I want to make sure nothing goes boom.

Reply:
The series is being retrieved from the application.
The value is compared against the charm in (at least) `application.SetSeries` and `application.SetCharm`, accommodating things like `ForceSeries` and the list of supported series in charm metadata.

So it is singular in this context and tells _what the application is doing with the charm_.

I just teased this out as best as I could glean it, so let me know if this approach is incorrect.